### ISSUE TITLE
Lab py files failing in the terminal

### ISSUE THREAD

**Issue by adamlimsft** [MicrosoftLearning/mslearn-azure-ml#96]
# Module: Run a training script as a command job in Azure Machine Learning
#### Step: 03  python train-model-parameters.py --training_data diabetes.csv  The script should successfully run and as a result, the output should show the accuracy and AUC of the trained model.

Description of issue:
train-classification-model.py and train-model-parameters.py script both fail when running in the terminal. 

Repro steps:

1. Run the py script in terminal. Other MTTs also confirmed that they run into same error.
![image](https://github.com/user-attachments/assets/1a9695da-1de4-43ee-94b0-489757c09867)

**Comment by afelix-95** [comment]
@adamlimsft, @madiepev I tested the lab and it is an issue of outdated library in the Anaconda environment. It's a product issue but we can manually fix it by adding a repository with the latest versions of the library, updating the package list and upgrading the `libstdc++6` package:

``` bash
sudo add-apt-repository ppa:ubuntu-toolchain-r/test
sudo apt-get update
sudo apt-get upgrade libstdc++6
```

**Comment by Jason-Microsoft-MTT** [comment]
@afelix-95 nice catch, I found that solution myself (thanks to copilot). Only addition I will make is that to make sure you are executing these commands in the terminal when you are uninstalling/reinstalling the azure ai-ml package (Task: Clone the lab materials  Step 6)  and it does take about 5 minutes for these updates to happen.

**Comment by chris-utter** [comment]
Thanks @afelix-95 I ran into this today (9/11/24). I recommend adding these steps to the instructions somewhere until the environment is updated.

**Comment by chris-utter** [comment]
I received the following error when running the first command line (sudo add-apt-repository ppa:ubuntu-toolchain-r/test):

 The following signatures were invalid: EXPKEYSIG 544B7F63BF9E4D5F Tensorflow Serving Developer (Tensorflow Serving APT repository key) <tensorflow-serving-dev@googlegroups.com>
Hit:20 http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu focal InRelease
Fetched 1484 B in 2s (739 B/s)
Reading package lists... Done
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://storage.googleapis.com/tensorflow-serving-apt stable InRelease: The following signatures were invalid: EXPKEYSIG 544B7F63BF9E4D5F Tensorflow Serving Developer (Tensorflow Serving APT repository key) <tensorflow-serving-dev@googlegroups.com>
W: Failed to fetch https://storage.googleapis.com/tensorflow-serving-apt/dists/stable/InRelease  The following signatures were invalid: EXPKEYSIG 544B7F63BF9E4D5F Tensorflow Serving Developer (Tensorflow Serving APT repository key) <tensorflow-serving-dev@googlegroups.com>
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by afelix-95** [comment]
@chris-utter you can ignore those errors as the required files for updating the library are downloaded anyway. You can still run the other commands and they'll execute without any issues.
